test/js-array/postcss.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function (ctx) {
+  return {
+    plugins: [
+      require('postcss-import')(),
+      // simulate the case of require('doiuse') without actually requiring it
+      {
+        postcss: require('postcss-nested').postcss
```

Comment:
`postcss-nested` actually _relies_ on the `i = i.postcss` behaviour of `Processor.prototype.normalize`. It's just that, while `postcss-nested`'s `.postcss` is a property attached to a `Function`, `doiuse`'s `.postcss` is inside a plain object.